src/main/kotlin/uk/co/craigbass/pratura/domain/Currency.kt

```diff
@@ -0,0 +1,17 @@
+package uk.co.craigbass.pratura.domain
+
+import org.joda.money.*
+import org.joda.money.format.MoneyFormatterBuilder
+import java.math.BigDecimal
+import java.util.*
+
+class Currency(val currency: String, val country: String, val language: String) {
```

Comment:
This class is also a bit odd in that why not just have one format method that takes Money and Locale